experimental/packages/opentelemetry-exporter-prometheus/test/PrometheusSerializer.test.ts

```diff
@@ -265,14 +265,14 @@ describe('PrometheusSerializer', () => {
   });
 
   describe('validate against metric conventions', () => {
-    async function getCounterResult(name: string, serializer: PrometheusSerializer) {
+    async function getCounterResult(name: string, unit: string | undefined = undefined, serializer: PrometheusSerializer, exportAll = false) {
```

Comment:
Alternating required and optional parameters is not very usual. Maybe think about making the function take an options object? `getCounterResult(name: string, serializer: PrometheusSerializer, options: Partial<{ unit: string, exportAll: boolean }> = {})`

Reply:
Will do